examples/docker/example-package.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
There is a reason why this file is not named ```package.json```. If I rename it as you propose, when building the theia-trace-extension application from the source of this repo (I mean if you try to checkout this branc/pull-request, then you locally on your machine run ```cd theia-trace-extension && yarn```), it will create a lockfile that contains the dependency to  ``` "theia-traceviewer": "next" ```.

This probably happens because the package manager is looking recursively to all package.json in the project (including the ones in the example folder) and using them to build the dependency tree. I am not sure if this is expected or not, but I do not want that an "example" is going to pollute the dependency tree of the main project. Hence the renaming.

Please let me know if you have comments about this, but I think that fixing the issue (or feature?) of dependencies / ignoring or not the package.json files under the example folder should be kept in a separate change because this is not connected to the "dockerizing" work that this PR is targeting.

Reply:
Ah, LGTM then too; thanks for clarifying. I'd add a note about this in the commit message though, to capture this important info in git history, up to you. Or in the README, at will. (Maybe the latter would be closer to us potential users of this.)

Reply:
Done, please resolve if satisfied